compute/admin_guide/authentication/oidc.adoc

```diff
@@ -116,12 +115,92 @@ Configure Prisma Cloud.
 
 .. Click *Save*.
 
+[.task]
+=== Azure Active Directory (AD)
+
+To integrate with Azure Active Directory (AD), you must register Prisma Cloud as an Open ID Connect (OIDC) application in Azure and configure Azure AD as an identity provider in Prisma Cloud.
+
+[.procedure]
+
+. Go to https://portal.azure.com/#home[your Azure console].
+
+. Find the Azure AD service.
```

Comment:
Azure AD -> Azure Active Directory

Reply:
@Tomerit We don't need to spell out the abbreviation here because it has been spelled out on first mention just above.